packages/perseus/src/widgets/interactive-graphs/graphs/sinusoid.tsx

```diff
@@ -97,7 +91,7 @@ export const computeSine = function (
 
 export const getSinusoidCoefficients = (
     coords: ReadonlyArray<Coord>,
-): SineCoefficient | undefined => {
+): NamedSineCoefficient | undefined => {
```

Comment:
There were two `SineCoefficient` types and they were different, so I renamed one of them.